src/components/TableItem/TableItem.tsx

```diff
@@ -0,0 +1,80 @@
+import { Link, NavLink } from 'react-router-dom';
+import cn from 'classnames';
+import React from 'react';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+  selectedUser: string | undefined;
+};
+
+const createLink = (base: string, salt: number) => {
+  return `${base.toLowerCase().replace(/\s/g, '-')}-${salt}`;
+};
+
+export const TableItem: React.FC<Props> = ({ person, selectedUser }) => {
+  const {
+    name, sex, born, died, mother, father, motherName, fatherName,
+  } = person;
+
+  const personLink = createLink(name, born);
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn(
+        { 'has-background-warning': personLink === selectedUser },
+      )}
+    >
+      <td>
+        <NavLink
+          to={`../${personLink}`}
+          className={cn(
+            { 'has-text-danger': sex === 'f' },
```

Comment:
Create a enum for magic words and use them everywhere
```suggestion
            { 'has-text-danger': sex === 'f' },
```